src/frontend/admin/src/components/presentational/table/TableComponent.tsx

```diff
@@ -200,6 +205,23 @@ export function TableComponent<T extends GridValidRowModel>({
             return props.rowHeight ?? 40;
           }}
           paginationMode={paginationMode}
+          sortingMode={sortingMode}
+          onSortModelChange={(sortModel) => {
+            const query = { ...router.query };
+
+            if (sortModel[0]) {
+              const filterName = sortModel[0].field;
+              const order = sortModel[0].sort === "asc" ? "" : "-";
+              const ordering = `${order}${filterName}`;
+              props.onFilter?.({ ordering });
+              query.ordering = ordering;
+            } else {
+              props.onFilter?.({ ordering: "" });
```

Comment:
IMO it is a little more explicit
```suggestion
              props.onFilter?.({ ordering: undefined });
```